google-maps/e2e-tests/src/pages/Markers/MarkerCustomizations.tsx

```diff
@@ -0,0 +1,389 @@
+          iconSize: {
+            width: 30,
+            height: 30,
+          },
+        },
+        {
+          coordinate: {
+            lat: 47.3,
+            lng: -122.46,
+          },
+          title: 'Title 3',
+          snippet: 'Snippet 3',
+          iconUrl: 'assets/icon/pin.png',
+          iconSize: {
+            width: 30,
+            height: 30,
+          },
+        },
+        {
+          coordinate: {
+            lat: 47.2,
+            lng: -122.23,
+          },
+          title: 'Title 4',
+          snippet: 'Snippet 4',
+          iconUrl: 'assets/icon/pin.png',
+          iconSize: {
+            width: 30,
+            height: 30,
+          },
+        },
+      ];
+
+      const ids = await map.addMarkers(markers);
+      map.setCamera({
+        animate: true,
+        coordinate: {
+          lat: 47.6,
+          lng: -122.33,
+        },
+        zoom: 7,
+      });
+      setMarkerIds(ids);
+      setCommandOutput(`${ids.length} image markers added`);
+    } catch (err: any) {
+      setCommandOutput(err.message);
+    }
+  };
+
+  const addMultipleColorMarkers = async () => {
+    try {
+      if (!map) {
+        throw new Error('map not created');
+      }
+      const markers: Marker[] = [
+        {
+          coordinate: {
+            lat: 47.6,
+            lng: -122.33,
+          },
+          title: 'Title 1',
+          snippet: 'Snippet 1',
+          tintColor: {
+            r: getRandom(),
+            g: getRandom(),
+            b: getRandom(),
+            a: 1,
+          },
+        },
+        {
+          coordinate: {
+            lat: 47.6,
+            lng: -122.46,
+          },
+          title: 'Title 2',
+          snippet: 'Snippet 2',
+          tintColor: {
+            r: getRandom(),
+            g: getRandom(),
+            b: getRandom(),
+            a: 1,
+          },
+        },
+        {
+          coordinate: {
+            lat: 47.3,
+            lng: -122.46,
+          },
+          title: 'Title 3',
+          snippet: 'Snippet 3',
+          tintColor: {
+            r: getRandom(),
+            g: getRandom(),
+            b: getRandom(),
+            a: 1,
+          },
+        },
+        {
+          coordinate: {
+            lat: 47.2,
+            lng: -122.23,
+          },
+          title: 'Title 4',
+          snippet: 'Snippet 4',
+          tintColor: {
+            r: getRandom(),
+            g: getRandom(),
+            b: getRandom(),
+            a: 1,
+          },
+        },
+      ];
+
+      const ids = await map.addMarkers(markers);
+      map.setCamera({
+        animate: true,
+        coordinate: {
+          lat: 47.6,
+          lng: -122.33,
+        },
+        zoom: 7,
+      });
+      setMarkerIds(ids);
+      setCommandOutput(`${ids.length} color markers added`);
+    } catch (err: any) {
+      setCommandOutput(err.message);
+    }
+  };
+
+  const removeAllMarkers = async () => {
```

Comment:
This only removes the last markers added, not all on the map. Not sure if it's necessary to fix, but would help being able to test all functions at once